### ISSUE TITLE
is there any way to check the training status

### ISSUE THREAD

**Issue by andy51002000** [RasaHQ/rasa_nlu#1352]
**Rasa NLU version**:
{"version": "0.10.5"}

**Operating system** (windows, osx, ...):
docker

**Issue**:

I sent my data to rasa and I got an error message below.

 /usr/local/lib/python2.7/site-packages/sklearn/metrics/classification.py:1113: sklearn.exceptions.UndefinedMetricWarning: F-score is ill-defined and being set to 0.0 in labels with no predicted samples.

It has been 8 hours and I didn't see other messages. I'm not sure if my training process is dead or not.  I'm wondering, is there any way to check the training status.

**Comment by amn41** [comment]
For the tensorflow backend you will get a progressbar, for sklearn there is an open issue to implement the same https://github.com/RasaHQ/rasa_nlu/issues/1114

How much training data do you have?

**Comment by andy51002000** [comment]
Thanks for your quick reply.

I have

- intent examples: 178955 (33 distinct intents)
- entity examples: 178907 (6 distinct entities)

My CPU information is 

CPU(s): 20
On-line CPU(s) list: 0-19
Thread(s) per core: 2
Core(s) per socket: 10
Socket(s): 1
NUMA node(s): 1
Vendor ID: GenuineIntel
CPU family: 6
Model: 85
Model name: Intel(R) Core(TM) i7-7900X CPU @ 3.30GHz
Stepping: 4
CPU MHz: 1599.984
CPU max MHz: 4500.0000
CPU min MHz: 1200.0000
BogoMIPS: 6624.59
Virtualization: VT-x
L1d cache: 32K
L1i cache: 32K
L2 cache: 1024K
L3 cache: 14080K
NUMA node0 CPU(s): 0-19

**Comment by amn41** [comment]
ok that will take a while :) one thing you can try is to subclass the `SklearnIntentClassifier`,

override this method: https://github.com/RasaHQ/rasa_nlu/blob/master/rasa_nlu/classifiers/sklearn_intent_classifier.py#L140

and replace it with the linearSVC http://scikit-learn.org/stable/modules/generated/sklearn.svm.LinearSVC.html

that should scale better to a large number of samples
